DevelopGitHub/GitHubExtension/GitHubExtension.Notes.Tests/TestsForRoutes/NoteRouteTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Net.Http;
+using GitHubExtension.Infrastructure.Constants;
+using GitHubExtension.Notes.Tests.TestsForRoutes.TestRoutesMappers;
+using GitHubExtension.Notes.WebApi.Controllers;
+using GitHubExtension.Notes.WebApi.ViewModels;
+using MvcRouteTester;
+using Xunit;
+
+namespace GitHubExtension.Notes.Tests.TestsForRoutes
+{
+    public class NoteRouteTests : TestNoteRoutesConfig
+    {
+        public NoteRouteTests() : base(RouteConstants.ApiNote)
+        {
+        }
+
+        [Fact]
+        public void GetNoteForCollaboratorRoute()
+        {
+            url = url.ForGetNoteForCollaborator();
+            var routes = GetRoutes();
```

Comment:
I managed to work correctly with routes from attributes only by calling GetRoutes method each time when url for a specific route is set and now I have to call it for each test.